src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Capacity.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.locationsinsideprison.jpa
+
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+
+@Entity
+class Capacity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
```

Comment:
Minor but you could make the ids not-nullable and initialise  to 0. Thats what weve done in other repos. Hibernate understands this value as 'unset'.